SideMenuSample/MainContainerModule/Presenter/MainContainerPresenter.swift

```diff
@@ -15,6 +15,7 @@ class MainContainerPresenter {
     private (set) var router: MainContainerRouter
     private (set) var interactor: MainContainerInteractorInput
     private (set) weak var view: MainContainerViewController?
+    fileprivate let swipeInteractionController = SwipeInteractionController(swipeDirection: .right, transitionThreshold: 0.3)
```

Comment:
please don't use literal. Is this `0.3` related to threshold in SlideInPresentationManager?